analysis/aws_rules_guardduty/aws_guardduty_high_sev_findings.yml

```diff
@@ -0,0 +1,60 @@
+AnalysisType: rule
+Filename: aws_guardduty_high_sev_findings.py
+PolicyID: AWS.GuardDuty.HighSeverityFinding
```

Comment:
Is this the "ruleId"? @nhakmiller

Reply:
Since there is a very basic one to one mapping of policy/rule fields, we currently just follow the naming conventions for policies. The `AnalysisType` at the top of the specification is what tells the backend to switch everything to rule fields. There is an open issue for this on the `panther-cli` side: https://github.com/panther-labs/panther-analysis/issues/19